### ISSUE TITLE
[WebGL] fill-color property also draws linear rings (clockwise?)

### ISSUE THREAD

**Issue by erwanlpfr** [openlayers/openlayers#14949]
**Describe the bug**
I am trying to convert all of my vectorial layers with polygons to the WebGL layers.
The issue, here, is that I use a world coordinate to draw a reversed zone (everything except hole).

**To Reproduce**
1. Go to [codesandbox](https://codesandbox.io/s/webgl-vector-layer-forked-98vtx9?file=/main.js)
2. Zoom to Rennes in France
3. See that 

**Expected behavior**
Before with vectorial style I had the similar behavior of geojson.io.
Now this.

![image](https://github.com/openlayers/openlayers/assets/113348625/642ad6a9-226d-4851-a4d9-04e9b4a9d17c)
![image](https://github.com/openlayers/openlayers/assets/113348625/805fad3b-cdd0-424b-a4b2-8d3f19fe9253)

The actual color I want in fill-color in actually good. But It looks like the color gets overlapped several times.

My investigation is that comes from the clockwise but wrong.

I used these two tools :
- https://geojsonlint.com/ to check the rules.
- https://github.com/mapbox/geojson-rewind to change the order.

NB : If I am not wrong, the usage of geojson-rewind must set the second parameter to false, to follow the 3.1.6 rule?

The result with WebGL
- when the geojson is valid, I have the issue.
- when the geosjon does not follow the rules, the issue disappears.

The result with my previous VectorLayer implem :
- when the geojson is valid, no issue.
- when the geojson is invalid, no issue.

I put comments inside the snippet to change the clockwise.

So, to sum up, it looks like a valid geojson (according to geojsonlint) is not well displayed but an invalid one is well displayed using the WebGL layer.

Let me know if I'm doing something wrong !

Thank you !

**Comment by jahow** [comment]
@erwanlpfr could you please by any chance confirm that the issue is now gone on the main branch? thanks a lot!

**Comment by erwanlpfr** [comment]
> @erwanlpfr could you please by any chance confirm that the issue is now gone on the main branch? thanks a lot!

Hello! After my first tests it looks working !
This week, I will give more time on it, and I'll be able to give a better feedback if necessary !

Thanks!
